src/js/components/Menu.js

```diff
@@ -164,6 +164,16 @@ class MenuDrop extends Component {
     return true;
   }
 
+  _renderNestedMenuLinks (menuItems) {
```

Comment:
From what I can recall, the point of this function is to only render the children or submenu items without returning or rendering the top level items. Hopefully that makes sense.

Without this function, nested menus in dropdown will not be rendered:

<img width="353" alt="screen shot 2016-02-19 at 10 28 40 am" src="https://cloud.githubusercontent.com/assets/7669/13184984/c87a270c-d6f3-11e5-977d-3956b5921f5b.png">